blocks/gallery-block/features/gallery/default.jsx

```diff
@@ -58,30 +72,89 @@ export const GalleryPresentation = ({
 
 	const interstitialClicks = parseInt(galleryCubeClicks, 10);
 
+	const galleryLength = contentElements.length;
+
 	return (
-		<Gallery
-			galleryElements={contentElements}
-			resizerURL={resizerURL}
-			ansId={id}
-			ansHeadline={headlines?.basic ? headlines.basic : ""}
-			expandPhrase={phrases.t("global.gallery-expand-button")}
+		<Carousel
+			id={id}
+			className={BLOCK_CLASS_NAME}
+			showLabel
+			label={headlines?.basic ? headlines.basic : ""}
+			slidesToShow={1}
+			showAdditionalSlideControls
+			pageCountPhrase={
+				/* istanbul ignore next */ (current, total) =>
+					phrases.t("global.gallery-page-count-text", { current, total })
+			}
+			enableAutoplay
+			startAutoplayIcon={<Icon name="Play" />}
+			startAutoplayText={phrases.t("global.gallery-autoplay-button")}
+			stopAutoplayIcon={<Icon name="Pause" />}
+			stopAutoplayText={phrases.t("global.gallery-pause-autoplay-button")}
 			autoplayPhraseLabels={{
 				start: phrases.t("global.gallery-autoplay-label-start"),
 				stop: phrases.t("global.gallery-autoplay-label-stop"),
 			}}
-			autoplayPhrase={phrases.t("global.gallery-autoplay-button")}
-			pausePhrase={phrases.t("global.gallery-pause-autoplay-button")}
-			controlsFont={getThemeStyle(arcSite)["primary-font-family"]}
-			pageCountPhrase={
-				/* istanbul ignore next */ (current, total) =>
-					phrases.t("global.gallery-page-count-text", { current, total })
+			enableFullScreen
+			fullScreenShowButton={
+				<button type="button">
+					<Icon name="Fullscreen" />
+					{phrases.t("global.gallery-expand-button")}
+				</button>
+			}
+			fullScreenMinimizeButton={
+				<button type="button">
```

Comment:
`button` requires an `id` - and an `id` must be unique

Reply:
updated

Reply:
will update article body as well

Reply:
https://github.com/WPMedia/arc-themes-blocks/pull/1497

Reply:
I miss spoke on the `id` value here - the id's of the buttons should match the `id` prop of the `Carousel`

Reply:
the aria controls are on the button crucially

Reply:
![Screen Shot 2022-08-31 at 10 15 55](https://user-images.githubusercontent.com/5950956/187714905-bc094090-fde4-4400-9668-a26987942ea6.png)

Reply:
we shouldn't have duplicate ids though

Reply:
The Caousel.Button "id" is not the HTML id property... it's poorly named I think.